test/parallel/test-dgram-send-empty-array.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+
+const client = dgram.createSocket('udp4');
+
+const timer = setTimeout(function() {
+  throw new Error('Timeout');
+}, common.platformTimeout(200));
```

Comment:
I understand that: it's UDP.
But I think these kind timers (`the test is taking longer than expected so we throw`) should only exist when testing timing related features (timers, connection timers, etc.) otherwise the test runner should catch them. I think @Trott made some good points here: https://github.com/nodejs/testing/issues/27#issuecomment-199995304.

Anyway, I'm fine either way.